VAMobile/src/screens/HealthScreen/Appointments/UpcomingAppointments/UpcomingAppointmentDetails.tsx

```diff
@@ -113,6 +114,12 @@ const UpcomingAppointmentDetails: FC<UpcomingAppointmentDetailsProps> = ({ route
     }
   }, [appointmentCancellationStatus, dispatch, navigation])
 
+  useEffect(() => {
+    if (!screenError && appointmentNotFound) {
+      logAnalyticsEvent(Events.vama_appt_deep_link_fail(vetextID))
```

Comment:
@dumathane Here's where we track which push notifications are tapped:

https://github.com/department-of-veterans-affairs/va-mobile-app/blob/feature/7490-theo-appointment-deep-link-analytics/VAMobile/src/components/NotificationManager/NotificationManager.tsx#L54

So we'll be able to compared push notification with the `notification.payload.url` that start with `vamobile/appointments/` to the number of people that have the`vama_appt_deep_link_fail` even triggered